Lib/fontParts/base/layer.py

```diff
@@ -133,7 +133,7 @@ def __delitem__(self, name):
         """
         name = normalizers.normalizeGlyphName(name)
         if name not in self:
-            raise ValueError("No glyph with the name '%s' exists." % name)
+            raise KeyError("No glyph with the name '%s' exists." % name)
```

Comment:
Maybe use the opportunity to make both error messages consistent. I like `"No glyph named '%s'."` better.

Reply:
Agree!